style/ruby/.rubocop.yml

```diff
@@ -0,0 +1,15 @@
+AllCops:
+  Exclude:
+    - 'Gemfile'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'spec/spec_helper.rb'
```

Comment:
Rubocop found good errors in Argo `config` and `spec_helper` for me.  I only got false positives from the cap scripts where the cap DSL context is not understood.